### DISCUSSION TITLE
Angular v13.0 and tailwindcss v3.0 integration issue

### DISCUSSION THREAD

**Discussion by abhinav-sriraj** [tailwindlabs/tailwindcss#10225] [Help]
Hi,
I've followed all the steps mentioned in the official documentation (https://tailwindcss.com/docs/guides/angular) and I successfully added Tailwindcss V3.0 to my Angular v13.0 project and I can see tailwind classes/styles being applied successfully.

The issue I'm currently facing is,
Whenever I add new tailwind styles, they don't show after saving (angular live - reload)
But when I stop the angular project and reload, the styles get applied successfully.. 
Now I can't keep killing and restarting the port whenever I add a new tailwind style during development.

Please help me on this,
Thank you

**Comment by ProPhycient5** [comment]
@abhinav-sriraj you can try deleting node modules and re-install `npm` and you can also run `build` if necessary. Please do check if it helps.

**Reply by abhinav-sriraj** [reply]
Hey @ProPhycient5 , 
Thanks for your reply but it didn't work.
